raven-app/src/components/feature/files/FilesSharedInChannel.tsx

```diff
@@ -32,11 +34,17 @@ export const FilesSharedInChannel = () => {
         borderColor: colorMode === 'light' ? 'gray.200' : 'gray.600'
     }
 
+    const handleClick = () => {
```

Comment:
why add this in a new function? just call onGlobalSearchModalOpen directly on button click?